kb/python-sandbox-environment.md

```diff
@@ -0,0 +1,136 @@
+---
+slug: python-sandbox-environment
+title: Python sandbox environment
+tags:
+  - kb-article
+date: 2022-11-18T00:00:00Z
+---
+
+![Python sandbox](static/img/../../../static/img/python-sandbox.svg)
+
+The Temporal Python SDK allows you to run Workflow code in a sandbox environment to help prevent non-determinism errors in your application.
```

Comment:
```suggestion
The Temporal Python SDK allows you to run Workflow code in a sandbox environment to help prevent non-determinism errors in your application.
```

allows -> enables

But since this is by default - it is more like "has an opt-out Sandbox"